paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -18,13 +17,12 @@ import com.stripe.android.utils.AnimationConstants
 import javax.inject.Inject
 
 /**
- * 🏗 This feature is under construction 🏗
+ * 🏗 This feature is in private beta and could change 🏗
```

Comment:
Can we also update the badge in the example app and make it link to the relevant docs page?